mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/services/SanityService.kt

```diff
@@ -0,0 +1,52 @@
+package no.nav.mulighetsrommet.api.services
+
+import io.ktor.client.*
+import io.ktor.client.call.*
+import io.ktor.client.engine.cio.*
+import io.ktor.client.plugins.*
+import io.ktor.client.plugins.contentnegotiation.*
+import io.ktor.client.request.*
+import io.ktor.client.statement.*
+import io.ktor.http.*
+import io.ktor.serialization.kotlinx.json.*
+import no.nav.mulighetsrommet.api.AppConfig
+import org.slf4j.LoggerFactory
+import java.text.SimpleDateFormat
+import java.util.*
+
+class SanityService(appConfig: AppConfig) {
```

Comment:
Kan evt. endre denne til å ta inn `SanityConfig` eller `projectId` og `authToken` i stedet for `AppConfig`, så blir det litt tydligere hvilke input som trengs av klassen

Reply:
Godt poeng, jeg skal fikse det